### PR TITLE
Update app-sdk and rewrite webhook to new api

### PR DESCRIPTION
**Pull request by lkostrowski**
Wait for sdk 0.19.0

### CODE REVIEW

**Review by kzuraw — APPROVED** [review]

File: src/pages/api/webhooks/order-created.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  const response = await sendSlackMessage(webhookUrl, {
+    saleorDomain: authData.domain,
+    order: payload.order,
+  });
+
+  if (response.status !== 200) {
+    const errorMessage = await response.text();
+    console.error(`Slack API responded with code ${response.status}: ${errorMessage}`);
+
+    return Response.InternalServerError({
```
Comment (krzysztofwolski) [inline src/pages/api/webhooks/order-created.ts]: Are retes responses compatible inside the next handlers? 🤔

Comment (lkostrowski) [reply src/pages/api/webhooks/order-created.ts]: probably not, I copied it from original code, will change

**Review by krzysztofwolski — APPROVED** [review]
Lookin good!

**Comment by lkostrowski** [comment]
Closing in favor of https://github.com/saleor/saleor-app-slack/pull/27

**Pull request closed without merging**